src/components/Modeler.vue

```diff
@@ -4,7 +4,7 @@
       v-if="tooltipTarget"
       :key="tooltipTarget.id"
       :target="getTooltipTarget"
-      :title="tooltipTitle"
+      :title="tooltipTitle || ''"
```

Comment:
The tooltip shouldn't render if an empty string is passed to it, so this can ca left as:
```
 :title="tooltipTitle"
```
And `hideTooltip ` can be removed.